Sources/Xcodeproj/pbxproj().swift

```diff
@@ -353,6 +414,12 @@ public func pbxproj(srcroot: AbsolutePath, projectRoot: AbsolutePath, xcodeprojP
     print("}")
 }
 
+extension AbsolutePath {
+    func groupReference(srcroot base: AbsolutePath) -> String {
```

Comment:
It doesn't seem right to make this an extension of path, since the functionality being added is Xcode-specific.  I am not yet 100% sure what the Swifty way to do this is, but extending a primitive type like `AbsolutePath` doesn't seem quite right.  At the least it should be `fileprivate` to limit exposure of this member function.